nix/pkgs/android-sdk/shell.nix

```diff
@@ -10,11 +10,10 @@ mkShell {
     export ANDROID_NDK_ROOT="${androidPkgs.ndk}"
     export ANDROID_SDK_ROOT="$ANDROID_HOME"
 
-    export PATH="$ANDROID_HOME/bin:$PATH"
     export PATH="$ANDROID_NDK_ROOT:$PATH"
     export PATH="$ANDROID_SDK_ROOT/tools:$PATH"
     export PATH="$ANDROID_SDK_ROOT/tools/bin:$PATH"
+    export PATH="$ANDROID_SDK_ROOT/cmdline-tools/9.0/bin:$PATH"
```

Comment:
Hardcoding the same version is not great.

I thought we could get it from `compose` object, like you can for other sub-packages:
```
 > make nix-repl
nix repl default.nix
warning: future versions of Nix will require using `--file` to load a file
Welcome to Nix 2.14.1. Type :? for help.

Loading installable ''...
Added 4 variables.
nix-repl> pkgs.androidEnvCustom.compose.platform-tools.version
"33.0.3"

nix-repl> pkgs.androidEnvCustom.compose.ndk-bundle.version
"22.1.7171670"
```
I found this:
```
nix-repl> :p pkgs.androidEnvCustom.compose.androidsdk.drvAttrs.version
"9.0"
```
But I haven't checked if this is exactly the same value and can be trusted. You'd have to check the derivation.

We could also just use `echo` and a wildcard:
```sh
 > echo /tmp/test/*/bin | tr ' ' ':'
/tmp/test/1.1/bin:/tmp/test/1.2/bin
```